backend/app/schemas.py

```
@@ -19,18 +19,24 @@
 
 class ResetPasswordRequest(BaseModel):
     token: str
-    new_password: str
-
+    new_password: str = Field(..., min_length=8, max_length=50, description="Password must be between 8-50 characters.")
+    
+    @field_validator("new_password")
+    @classmethod
+    def validate_password(cls, value):
```

Ruff (0.8.2)

Missing return type annotation for classmethod `validate_password`

(ANN206)